src/components/map/controls/layer-toggle/modal/component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Icon from 'components/icon';
+import Switch from 'components/switch';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const LayersToggleModalComponent = ({ handleCloseModal, checked, setChecked }) => {
+  return (
+    <div className={styles.toggleLayerModal}>
+      <div className={styles.header}>
+        <span className={styles.title}>ADD LAYERS TO THE MAP</span>
+        <button onClick={handleCloseModal}>
+          <Icon name="icon-close" />
+        </button>
+      </div>
+      <div className={styles.content}>
+        <Switch
+          value="layer-1"
+          checked={checked}
+          handleChange={() => setChecked(!checked)}
+          name="Land use"
+        />
+        <Switch
+          value="layer-2"
+          checked={checked}
+          handleChange={() => setChecked(!checked)}
+          name="Plantations"
+        />
+        <div className={styles.spacer} />
+        <Switch
+          value="layer-3"
+          checked={checked}
+          handleChange={() => setChecked(!checked)}
+          name="Labels"
+        />
+        <Switch
+          value="layer-4"
+          checked={checked}
+          handleChange={() => setChecked(!checked)}
```

Comment:
you're using `setChecked(!checked)` in couple of places, I'd suggest you to move it to `toggleChecked` function and use that one instead
```
handleChange={toggleChecked}
```

Reply:
I wouldn't bother about this, because those four switches are just placeholders, I'm going to rewrite this component with real logic soon :)

Reply:
👍